src/extension/background-script/liquid/index.ts

```diff
@@ -0,0 +1,28 @@
+import * as secp256k1 from "@noble/secp256k1";
+
+class Liquid {
+  privateKey: string;
+
+  constructor(privateKey: string) {
+    this.privateKey = privateKey;
+  }
+
+  getPublicKey() {
+    const publicKey = secp256k1.schnorr.getPublicKey(
+      secp256k1.utils.hexToBytes(this.privateKey)
+    );
+    const publicKeyHex = secp256k1.utils.bytesToHex(publicKey);
+    return publicKeyHex;
+  }
+
+  async signSchnorr(sigHash: string): Promise<string> {
```

Comment:
It would be great to add signSchnorr for webbtc. I can add it once this PR is merged